forest/src/subcommand.rs

```diff
@@ -1,10 +1,63 @@
 // Copyright 2020 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use super::cli::Subcommand;
+use auth::{create_token, ADMIN, JWT_IDENTIFIER};
+use rpassword::read_password;
+use rpc_client::API_INFO;
+use std::path::PathBuf;
+use wallet::{KeyStore, KeyStoreConfig, ENCRYPTED_KEYSTORE_NAME};
+
+use super::cli::{Config, Subcommand};
 
 /// Process CLI subcommand
-pub(super) async fn process(command: Subcommand) {
+pub(super) async fn process(command: Subcommand, config: Config) {
+    let token = match config.rpc_token.to_owned() {
+        Some(token) => token,
+        // If no token argument is passed or configured, attempt to load it from the local keystore
+        None => {
+            let keystore = if config.encrypt_keystore {
+                loop {
+                    println!("keystore passphrase: ");
```

Comment:
Can we use a full sentence here and capitalize on new lines? `Enter the keystore passphrase: `